### PR TITLE
Split crm_mon.c up into smaller files.

### PR DESCRIPTION
**Pull request by clumens**
This introduces crm_mon.h, crm_mon_print.c, and crm_mon_runtime.c.  The first file should be obvious.  The second is all the print-related functions, exposing only the top-level ones as API.  The third is stuff needed by more than one crm_mon source file.

### CODE REVIEW

File: tools/crm_mon.c [hunk 1]
```diff
@@ -2009,10 +2012,10 @@ print_node_attribute(gpointer name, gpointer user_data)
     value = pe_node_attribute_raw(data->node, name);
 
     /* Print attribute name and value */
-    switch (output_format) {
+    switch (data->fmt) {
         case mon_output_plain:
         case mon_output_console:
-            print_as("    + %-32s\t: %-10s", (char *)name, value);
+            print_as(output_format, "    + %-32s\t: %-10s", (char *)name, value);
```
Comment (kgaillot) [inline tools/crm_mon.c]: Do you mean data->fmt here and below?

File: tools/crm_mon_runtime.c [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+        count += count_resources(data_set, gIter->data);
+    }
+    return count;
+}
+
+void
+crm_mon_get_parameters(resource_t *rsc, pe_working_set_t * data_set)
+{
+    get_rsc_attributes(rsc->parameters, rsc, NULL, data_set);
+    crm_trace("Beekhof: unpacked params for %s (%d)", rsc->id, g_hash_table_size(rsc->parameters));
```
Comment (kgaillot) [inline tools/crm_mon_runtime.c]: We can drop this trace message, it was just for debugging at the time :)

File: tools/crm_mon.c (hunk 1 above)
Comment (clumens) [reply tools/crm_mon.c]: Yes and in fact I fix this in a later patch, but it's worth fixing in the earlier patch.

**Comment by clumens** [comment]
> Getting to the fun parts soon!

Yeah I've got plenty of patches that start implementing the fun stuff, too.  Probably after this PR I'll have one that puts in the basics of formatted output and also does a bit more rearranging, and then it's on to actual formatted output.  I'll also double check that no more of my rearranging patches can go into the PR.

**Pull request merged**